### ISSUE TITLE
Title bar renders incorrectly with border pixel config

### ISSUE THREAD

**Issue by minus7** [swaywm/sway#821]
To reproduce:
Configure `border pixel 10` or `new_window pixel 10`, then spawn (at least) one window, then split that window's container, then switch it to tabbed layout.

No space is allocated for the title bar (that is otherwise disabled with said config but should appear with said steps) and it is drawn behind the window contents.
![screenshot_2016-08-02_22-29-56](https://cloud.githubusercontent.com/assets/419104/17344010/d1bbd22c-5900-11e6-9b79-afe377840557.png)

**Comment by SirCmpwn** [comment]
Cannot reproduce. Still an issue?

**Comment by minus7** [comment]
Issue still persists:
- Start sway (in X? rev de79fee) with folowing config:
```
new_window pixel 10
exec xterm
```
- Open empty workspace
- Start terminal
- run `swaymsg 'split v'`
- run `swaymsg 'layout tabbed'`

**Comment by thejan2009** [comment]
> run swaymsg 'layout tabbed' 

@minus7 You said that the title bar should appear after this step. I tried the same sequence of commands in i3 and there was no title bar as long as only one window was open. The titlebar part that you see on the left top side of screenshot, behind the terminal window, is "just" a residue of previous drawing of the borders (that was also reported in #745). 

The  only problem I noticed here is that the top border didn't get drawn - the darker grey colour in the top border is actually #383838 solid background.

![2016-11-16-215411_swaygrab](https://cloud.githubusercontent.com/assets/1621134/20365544/83879d3a-ac47-11e6-9108-4e0794522fb6.png)
